securedrop/i18n.py

```diff
@@ -72,3 +75,47 @@ def get_locale():
         return locale
     else:
         return getattr(config, 'DEFAULT_LOCALE', 'en_US')
+
+
+def _get_supported_locales(locales, supported, default_locale):
+    """Return SUPPORTED_LOCALES from config.py. It is missing return all
+    locales found in the translations directory.
+
+    """
+
+    if not supported:
+        return sorted(locales)
+    unsupported = set(supported) - set(locales)
+    if unsupported:
+        raise Exception("config.py SUPPORTED_LOCALES contains {} which is not "
+                        "among the locales found in the {} "
+                        "directory: {}".format(list(unsupported),
+                                               babel.translation_directories,
+                                               locales))
+    if default_locale and default_locale not in supported:
+        raise Exception("config.py SUPPORTED_LOCALES contains {} "
```

Comment:
We should make this a `LocaleNotFound` exception or something similar so we can catch it with higher specificity in the unit test.

Reply:
Good idea, done !